### DISCUSSION TITLE
Corsair commander plugin?

### DISCUSSION THREAD

**Discussion by vladdeda** [Rem0o/FanControl.Releases#1264] [Q&A]
Anyone know what happened to the coraid plugin for commander and commander pro? It just disappeared and the Fan Control software no longer references it. :/ I have 18 corsair fans on 3 commander devices. One commander and 2 commander pro and I HATE using Icue.

**Comment by Fr0stX76** [comment]
See this: https://github.com/Rem0o/FanControl.Releases/discussions/1226

**Comment by EvanMulawski** [comment]
A replacement for that plugin is now available: https://github.com/EvanMulawski/FanControl.CorsairLink
